updater/updater.go

```diff
@@ -168,17 +177,32 @@ func run() error {
 		log.Info("finished deploy of system app")
 	}
 
-	// deploy all other apps
+	// deploy all other apps (including provisioned Redis apps)
 	apps, err := client.AppList()
 	if err != nil {
 		log.Error("error getting apps", "err", err)
 		return err
 	}
 	for _, app := range apps {
+		log := log.New("name", app.Name)
+
+		if strings.HasPrefix(app.Name, "redis-") {
```

Comment:
We should check this against some other piece of information like the resources table or the image URI, because users might have apps that start with `redis-`.